sys/reaper/reaper_utils_unix.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
The build-tag (`_linux.go`) didn't seem to match where it was used; given that this was used in `sys/reaper/reaper_unix.go`, which has `!windows`, I changed this to match.

Perhaps both need to be changed to `_linux.go` ?